### ISSUE TITLE
No encuentra ningún anime

### ISSUE THREAD

**Issue by jhanc97** [JuanM04/animu#2]
No encuentra nada.

**Comment by JuanM04** [comment]
Sí, ya no funciona. Cloudflare se actualizó y se me es imposible obtener datos de AnimeFLV

**Comment by jhanc97** [comment]
@JuanM04  lo provee nuevamente, y si encontró. Pero ahora al acceder a un video la pantalla queda en blanco sobre el contenido y no reproduce. Que podría ser?

**Comment by jhanc97** [comment]
![Screenshot_20201106_180750_com juanm04 animu](https://user-images.githubusercontent.com/73450591/98422925-0735a100-205b-11eb-823f-e17fb290c98f.jpg)

**Comment by JuanM04** [comment]
Si sabés algo de Flutter, te invito a qué investigues qué sucede. No te prometo que encuentre una solución pronto, porque hace tiempo que no hago nada para este proyecto.

**Comment by JuanM04** [comment]
> Sí, ya no funciona. Cloudflare se actualizó y se me es imposible obtener datos de AnimeFLV

Efectivamente, dejó de funcionar. Empecé a hacer un par de pedidos y se trabó

**Comment by xellDart** [comment]
Yo tengo un scrapper que hace bypass a Cloudflare, pero funciona en en servidor, si gustan podemos aliarnos y ver que hacemos... soy desarrollador de flutter

**Comment by JuanM04** [comment]
@xellDart me interesa tu propuesta. Mi email está en mi perfil, pero si usas Discord sería más sencillo
